ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -112,6 +115,16 @@ const styles = reactStyles({
 
 const pageId = 'workspace';
 
+const PublishButtonToolTipText = (isWorkspaceOwner: boolean) => {
+  let toolTip;
+  if (isWorkspaceOwner) {
+    toolTip = 'Contact Support to unpublish';
+  } else {
+    toolTip = 'Only workspaces owners can publish community workspace.';
```

Comment:
the grammar is slightly off here.  We should change this to "Only workspace owners can publish community workspaces."

or maybe some alternative like "Publishing workspaces is restricted to workspace owners."